connexion/decorators/produces.py

```diff
@@ -29,6 +29,9 @@ def default(self, o):
 
         if isinstance(o, datetime.date):
             return o.isoformat()
+        
```

Comment:
White space here. Please remove it. That's why `flake8` is complaining in the TravisCI job.